tests/internal/e2e/test/wdio.conf.ts

```diff
@@ -362,21 +374,72 @@ export const config: Options.Testrunner = {
     }
 
     process.env.CLICAST_PATH = clicastExecutablePath;
-
-    const vectorcastDir = path.dirname(clicastExecutablePath);
-    process.env.VC_DIR = vectorcastDir;
-
+    vectorcastDir = path.dirname(clicastExecutablePath);
+    
     let clearScreenshots: string;
     clearScreenshots =
       process.platform == "win32" ? "del /s /q *.png" : "rm -rf *.png";
     await promisifiedExec(clearScreenshots);
 
-    const testInputVcastTutorial = path.join(
-      initialWorkdir,
-      "test",
-      "test_input",
-      "vcastTutorial"
-    );
+    const createCFG = `cd ${testInputVcastTutorial} && clicast -lc template GNU_CPP_X`;
+      await promisifiedExec(createCFG);
+
+      const requestTutorialPath = path.join(
+        vectorcastDir,
+        "examples",
+        "RequirementsGW",
+        "CSV_Requirements_For_Tutorial.csv"
+      );
+      const commandPrefix = `cd ${testInputVcastTutorial} && ${process.env.CLICAST_PATH.trimEnd()} -lc`;
+      const rgwPrepCommands = [
+        `${commandPrefix} option VCAST_REPOSITORY ${path.join(
+          initialWorkdir,
+          "test",
+          "vcastTutorial"
+        )}`,
+        `${commandPrefix} RGw INitialize`,
+        `${commandPrefix} Rgw Set Gateway CSV`,
+        `${commandPrefix} RGw Configure Set CSV csv_path ${requestTutorialPath}`,
+        `${commandPrefix} RGw Configure Set CSV use_attribute_filter 0`,
+        `${commandPrefix} RGw Configure Set CSV filter_attribute`,
+        `${commandPrefix} RGw Configure Set CSV filter_attribute_value `,
+        `${commandPrefix} RGw Configure Set CSV id_attribute ID`,
+        `${commandPrefix} RGw Configure Set CSV key_attribute Key`,
+        `${commandPrefix} RGw Configure Set CSV title_attribute Title `,
+        `${commandPrefix} RGw Configure Set CSV description_attribute Description `,
+        `${commandPrefix} RGw Import`,
+      ];
+      for (const rgwPrepCommand of rgwPrepCommands) {
+        const { stdout, stderr } = await promisifiedExec(rgwPrepCommand);
+        if (stderr) {
+          console.log(stderr);
+          throw `Error when running ${rgwPrepCommand}`;
+        }
+
+        console.log(stdout);
+      }
+
+    
+    }else{
```

Comment:
Has this been `prettier`'d?

Reply:
Executed prettier in 11217bc161def3093067a7a263374993e8ab8a30.